extensions/avro-extensions/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

```diff
@@ -0,0 +1 @@
+io.druid.data.input.guice.AvroExtensionsModule
```

Comment:
nit: I would put AvroExtensionsModule inside io.druid.data.input.avro instead of creating a package just for that class.